api/articles/articles.go

```diff
@@ -96,3 +97,74 @@ func getArticle(id string) (*Article, error) {
 	}
 	return &article, err
 }
+
+func PostArticle(c *elesion.Context) {
+	// parse body (should be json)
+	body := Article{}
+	err := json.NewDecoder(c.Request.Body).Decode(&body)
+	if err != nil {
+		c.Status(http.StatusInternalServerError).Error(err.Error())
+		return
+	}
+	defer c.Request.Body.Close()
+
+	// insert into table
+	_, err = postArticle(body)
+	if err != nil {
+		c.Status(http.StatusInternalServerError).Error(err.Error())
```

Comment:
shall be 404? maybe the `user_id` is not exists, maybe the rows affected are greater than 1, and user in frontend is unnecessary to know what happened in backend. anyway there are logs in backend.